week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -7,11 +7,23 @@
 
   NOTE: don't include any element that is not a "family".
 */
+function findNamesWithA(names){
+  if(names.charCodeAt(0) === 65){
```

Comment:
65 does equal the letter `A` but you can just write "A" here to make this easier to read.

65 equals the letter A because that is what it is in the `ascii table` you can read more about that here.
http://www.asciitable.com/